tt_metal/impl/buffers/buffer.hpp

```diff
@@ -42,29 +42,29 @@ enum class ShardOrientation {
 
 
 struct ShardSpec {
-    CoreRangeSet shard_grid;
-    std::array<uint32_t, 2> shard_shape;
-    ShardOrientation shard_orientation = ShardOrientation::ROW_MAJOR;
+    CoreRangeSet grid;
+    std::array<uint32_t, 2> shape;
```

Comment:
can the shard shape be 3-D in the future?

Reply:
I think we might stick to 2d shards, same rank as a page (we only have 2d pages)